scene/resources/mesh.h

```diff
@@ -159,11 +159,43 @@ class Mesh : public Resource {
 	Size2i get_lightmap_size_hint() const;
 	void clear_cache() const;
 
-	typedef Vector<Vector<Face3>> (*ConvexDecompositionFunc)(const Vector<Face3> &p_faces, int p_max_convex_hulls);
+	struct ConvexDecompositionSettings {
+		enum Mode : int {
+			CONVEX_DECOMPOSITION_MODE_VOXEL = 0,
+			CONVEX_DECOMPOSITION_MODE_TETRAHEDRON
+		};
+
+		/// Maximum concavity. [Range: 0.0 -> 1.0]
+		real_t max_concavity = 1.0;
+		/// Controls the bias toward clipping along symmetry planes. [Range: 0.0 -> 1.0]
+		real_t symmetry_planes_clipping_bias = 0.05;
+		/// Controls the bias toward clipping along revolution axes. [Range: 0.0 -> 1.0]
+		real_t revolution_axex_clipping_bias = 0.05;
```

Comment:
> revolution_axex_clipping_bias

Can you check grammar and spelling?

Reply:
Thanks! Fixed. Not sure how to improve the doc however, I taken it from here: https://github.com/kmammou/v-hacd#parameters if you have any suggestion I'll be glad to edit it.